runtime/interpreter/value.go

```diff
@@ -815,6 +828,10 @@ func (v *StringValue) Transfer(
 	return v
 }
 
+func (v *StringValue) Clone(_ *Interpreter) Value {
+	return NewStringValue(v.Str)
```

Comment:
Not sure, do we need to copy the bytes of `v.Str` here?

Reply:
No need I think, because strings are immutable